File: java/utility/MakeContext.java
package voxgig.bluefintecsmerchantportalsdk.utility;

import java.util.Map;

import voxgig.bluefintecsmerchantportalsdk.core.Context;

final class MakeContext {

  private MakeContext() {}

  static Context makeContext(Map<String, Object> ctxmap, Context basectx) {
    return new Context(ctxmap, basectx);
  }
}

File: .sdk/tm/py/pkg/utility/make_context.py
# BluefinTecsMerchantPortal SDK utility: make_context

from projectname_sdk.core.context import BluefinTecsMerchantPortalContext


def make_context_util(ctxmap, basectx):
    return BluefinTecsMerchantPortalContext(ctxmap, basectx)
